uwblueprint/extend-a-family
# Jefferyhu/onboarding

## Notion ticket link
[Onboarding](https://www.notion.so/uwblueprintexecs/Setup-Onboarding-Task-d6e0cdeb69cf4bbf93a33c7302370550)

## Description
Completed the onboarding tasks!

## Steps to test
1. Click through the site pages
2. Ensure user can log in
3. Ensure user can add new users



## What should reviewers focus on?
* API Calls


## Checklist
